### DISCUSSION TITLE
Copy text out of chat?

### DISCUSSION THREAD

**Discussion by mchnvc** [ggozad/oterm#55] [Q&A] [answered]
Hi I was trying to do a simple copy and paste out of the response field. Basic selection doesn't work so was wondering if there was something about the textual interface that prevents it? Love what you've made so far though! Thanks!

**Comment by mchnvc** [comment, marked as answer]
NM .. switched to iterm and option select works!

**Comment by ggozad** [comment]
For future reference:

Handling selection in the terminal is quite tricky and not something I am willing to handle besides what `textual` already does.
You CAN copy even in terminals where you cannot select by clicking a "cell". This is obviously not amazing but it's what we can do :)

**Comment by mchnvc** [comment]
Thanks Yiorgis! That or maybe a button to copy to buffer. Either way great tool!

Sri Rao
Founder and Managing Partner
machineventures.vc
